pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter.go

```diff
@@ -88,8 +88,7 @@ func (admitter *MigrationCreateAdmitter) Admit(ar *v1beta1.AdmissionReview) *v1b
 	// are already in flight.
 	if vmi.Status.MigrationState != nil &&
 		string(vmi.Status.MigrationState.MigrationUID) != "" &&
-		!vmi.Status.MigrationState.Completed &&
-		!vmi.Status.MigrationState.Failed {
+		!vmi.Status.MigrationState.Completed {
```

Comment:
hm, I think this is more tricky... when I see correct, an aborted migration has Completed = false and Failed = false, but starting a new migration would be fine imho. My next idea was to check if EndTimestamp > StartTimestamp, but on a quick look I think EndTimeStamp is not set for failed migrations... @vladikr, can you help please?